concent_api/requirements.txt

```diff
@@ -24,3 +24,4 @@ pytest-xdist
 pytest-cov
 pytest-asyncio
 web3
+..//middleman_protocol
```

Comment:
Why two slashes? Is this a mistake or some special `pip` syntax?

Reply:
it is fixed with `-e ../`